docs/tasks/argo.rst

```diff
@@ -0,0 +1,253 @@
+Leveraging ActionSet Creation using Argo Cron Workflows
+------------------------------------------------------------
+
+Argo Workflows enable us to schedule operations. In the Kanister project,
+Argo Cron Workflows will be used to automate the creation of ActionSets to
+backup an application at regular intervals.
+
+To summarize, ActionSets are CRDs that are used to backup or restore an application.
+The controller present in the Kanister namespace looks for ActionSets to
+execute in the given namespace.
+
+In this tutorial you will schedule the creation of a backup actionset using
```

Comment:
I am seeing that at some places we have used `ActionSet` but at other places we have used `actionset`. I think we should chose one and stick to that. `ActionSet` is good I think.

Reply:
Done.